<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
    <remap from="/usb_cam/image_raw" to="lane_det/image" />
  </node>

  <node name="lane_seg" pkg="lane_det" type="hybridnets.py" />
  <node name="calibrate" pkg="lane_det" type="get_points.py" />
</launch>
